API.md

```diff
@@ -11,8 +11,8 @@
   - [`validate(value, schema, [options], [callback])`](#validatevalue-schema-options-callback)
   - [`compile(schema)`](#compileschema)
   - [`describe(schema)`](#describeschema)
-  - [`assert(value, schema, [message])`](#assertvalue-schema-message)
-  - [`attempt(value, schema, [message])`](#attemptvalue-schema-message)
+  - [`assert(value, schema, [message], [options])`](#assertvalue-schema-message-options)
```

Comment:
I think this is not clear that `options` can come in place of `message`. There is a notation for this case but I can't recall it right now.

Reply:
I was just following the formatting for `validate` which has a similar flexible contract with `options` and `callback` both being optional.

Reply:
True. Then I guess that consistency is a better option here, let's leave it as is :smiley: